bootloader/bootloader.go

```diff
@@ -192,6 +192,12 @@ type TrustedAssetsBootloader interface {
 	// the bootloader's rootdir that are measured in the boot process in the
 	// order of loading during the boot.
 	TrustedAssets() ([]string, error)
+
+	// RecoveryBootChain returns the chain of recovery mode boot files.
```

Comment:
// RecoveryBootChain returns the load chain for recovery modes.
// It should be called on a RoleRecovery bootloader.